src/ethereum_spec_tools/evm_tools/loaders/fixture_loader.py

```diff
@@ -0,0 +1,180 @@
+"""
+Defines Load class for loading json fixtures for the evm
+tools (t8n, b11r, etc.) as well as the execution specs
+testing framework.
+"""
+
+from abc import ABC, abstractmethod
+from typing import Any, Tuple
+
+from ethereum import rlp
+from ethereum.base_types import U256
+from ethereum.crypto.hash import Hash32
+from ethereum.utils.hexadecimal import (
+    hex_to_bytes,
+    hex_to_bytes8,
+    hex_to_bytes32,
+    hex_to_hash,
+    hex_to_u64,
+    hex_to_u256,
+    hex_to_uint,
+)
+
+from .fork_loader import ForkLoad
+from .transaction_loader import TransactionLoad
+
+
+class BaseLoad(ABC):
+    """Base class for loading json fixtures"""
+
+    @abstractmethod
+    def json_to_header(self, json_data: Any) -> Any:
+        """Converts json header data to a header object"""
+        pass
```

Comment:
```suggestion
        "Converts json header data to a header object"
        raise NotImplementedError()
```

Saw this style of abstract method in another Python project. What do you think?

Reply:
Makes sense. Updated